engine/access/rpc/backend/backend_transactions.go

```diff
@@ -205,7 +205,17 @@ func (b *backendTransactions) GetTransactionResult(
 	if txErr != nil {
 		if status.Code(txErr) == codes.NotFound {
 			// Tx not found. If we have historical Sporks setup, lets look through those as well
-			return b.getHistoricalTransactionResult(ctx, txID)
+			historicalTxResult, err := b.getHistoricalTransactionResult(ctx, txID)
+			if err != nil {
+				// if tx not found in old access nodes either, then assume that the tx was submitted to a different AN
+				// and return status as unknown
+				status := flow.TransactionStatusUnknown
+				return &access.TransactionResult{
+					Status:     status,
+					StatusCode: uint(status),
```

Comment:
What is the difference between `Status` and `StatusCode`?

Reply:
It is actually the same for now - https://github.com/onflow/flow/blob/master/docs/content/access-api.md#transaction-status - one is a number while other is the string representation.